### ISSUE TITLE
This is very different from the data-lessons version

### ISSUE THREAD

**Issue by jnothman** [LibraryCarpentry/lc-webscraping#26]
I'd be interested to know why all the changes (arguable improvements) to https://github.com/data-lessons/library-webscraping were not transferred here, which seems to have adopted, and then edited, an older version. Has there been consideration of bringing over any beneficial changes from data-lessons?

Apart from anything else, I think the Scraper tool used here is much less powerful than the state of the art in visual scraping systems: its limitation to single-page scrapes is especially problematic. Then getting students to understand site maps, fetching, etc, in order to write up a Python scraper, when they've previously only done single-page visual scraping, is a big jump for students who are not at home with coding.

**Comment by ostephens** [comment]
Agreed. My guess is that this was a mistake - only guessing but I wonder if the version here was copied from the master branch of the old repo rather than the gh-pages branch?
